### PR TITLE
fix: Correctly process IPs in ip_address_or_range

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: client/testing.go [hunk 1]
```diff
@@ -23,14 +23,21 @@ const (
 	FakeResourceGroup  = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test/providers/Microsoft.Storage/storageAccounts/cqprovidertest"
 )
 
+func clearIgnoreInTests(t *schema.Table) {
```
Comment (roneli) [inline client/testing.go:26]: nit: is this an issue in all providers? i.e this needs to be part of the SDK?

**Review by roneli — COMMENTED** [review]
LGTM, one minor nit

File: client/testing.go (hunk 1 above)
Comment (irmatov) [reply client/testing.go:26]: I think it is the same in all providers. IgnoreInTests is set to `false` only on top level table.

Not sure what needs to be moved to SDK. Just this little helper function?

Comment (irmatov) [reply client/testing.go]: There is a flag already that does just that on the SDK side. Updated the commit accordingly.

**Comment by irmatov** [comment]
Tests should pass after SDK update from https://github.com/cloudquery/cq-provider-sdk/pull/319

File: go.mod [hunk 2]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Azure/azure-sdk-for-go v61.6.0+incompatible
 	github.com/Azure/go-autorest/autorest v0.11.21
 	github.com/Azure/go-autorest/autorest/azure/auth v0.5.7
-	github.com/cloudquery/cq-provider-sdk v0.10.8
+	github.com/cloudquery/cq-provider-sdk v0.10.9
```
Comment (erezrokah) [inline go.mod:9]: A rebase should clear these changes from the PR I believe

**Review by erezrokah — APPROVED** [review]
This looks good, but I think it needs a rebase to remove the SDK changes

File: go.mod (hunk 2 above)
Comment (irmatov) [reply go.mod]: Updated

**Pull request merged**